includes/wf_crm_webform_postprocess.inc

```diff
@@ -1798,8 +1826,98 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
         $params['payment_instrument_id'] = key($defaultPaymentInstrument);
       }
     }
-    $result = wf_civicrm_api('contribution', 'create', $params);
-    $this->ent['contribution'][1]['id'] = $result['id'];
+
+    $createRecurringContribution = FALSE;
+    $numContributions = 1;
+    $numInstallments = wf_crm_aval($params, 'installments', 1, TRUE);
+    if ($numInstallments > 1 && $this->contributionIsPayLater) {
+      $createRecurringContribution = TRUE;
+      $numContributions = $numInstallments;
+    }
+
+    // Creating Recurring Contribution if needed.
+    $recurringContributionId = NULL;
```

Comment:
why do u call createRecurringContribution() here too, you already call it inside submitLivePayment(), I probably missing something but currently it feel like a duplicate to me.

Reply:
But submitLivePayment() is executed only for live payment processors (so not for Pay Later).